packages/builders/__tests__/util.test.ts

```diff
@@ -12,3 +12,18 @@ describe('validation', () => {
 		expect(isValidationEnabled()).toBeFalsy();
 	});
 });
+
+describe('normalizeArray', () => {
+	test('normalizes an array or array (when input is an array)', () => {
+		expect(normalizeArray([[1, 2, 3]])).toEqual([1, 2, 3]);
+	});
+
+	test('normalizes an array (when input is rest parameter)', () => {
+		expect(normalizeArray([1, 2, 3])).toEqual([1, 2, 3]);
+	});
+
+	test('always returns a clone', () => {
+		const arr = [1, 2, 3];
+		expect(normalizeArray([arr])).not.toBe(arr);
```

Comment:
I think this should check `toEqual(arr)` to make sure the output is the same as the input followed by the `not.toBe(arr)` to signal that the arrays are equal but they're not the same instance.